ledger/sandbox/src/main/scala/com/digitalasset/platform/index/StandaloneIndexServer.scala

```diff
@@ -80,8 +80,7 @@ class StandaloneIndexServer(
     writeService: WriteService) {
 
   // Name of this participant,
-  // TODO: Pass this info in command-line (See issue #2025)
-  val participantId: ParticipantId = Ref.LedgerString.assertFromString("postgress-participant")
+  val participantId: ParticipantId = Ref.LedgerString.assertFromString(config.participantId)
```

Comment:
We shouldn't assert that the user-provided `participantId` string is a valid `LedgerString`.